pdr_backend/models/prediction.py

```diff
@@ -11,6 +11,8 @@ def __init__(
         source,
         payout,
         user,
+        submittimestamp,
```

Comment:
Any reason this is not `submit_timestamp` ? (Snake case)

Reply:
done